astrokat/observe_main.py

```diff
@@ -370,12 +372,19 @@ def run_observation(opts, kat):
             if opts.all_up and (len(catalogue.filter(el_limit_deg=opts.horizon)) != len(catalogue)):
                 raise NotAllTargetsUpError('Not all targets are currently visible - '
                                            'please re-run the script with --visibility for information')
+        cal_tags = {'Polarisation': 'polcal',
+                    'Flux': 'fluxcal',
```

Comment:
Hi Ruby, can you also include the 'delaycal' tag. Thanks

Reply:
I added the "delaycal" tag, but I again advise that astronomers think about design when you put in requests. Delaycal means nothing for the observation and will lead to confusion. Ideally the processing software should be updated to reflect "gaincal" since that is the only complex gain calibration.